File: SIMauricio/pythonpoo/livro.py
class Livro:
    def __init__(self, titulo=None, autor=None, editora=None, anoPublicacao=None, preco=None):
        self.titulo = titulo
        self.autor = autor
        self.editora = editora
        self.anoPublicacao = anoPublicacao
        self.preco = preco

    def salvar(self):
        #Validar os dados e salvar
        print("Salvando dados do Livro...")
        return True
    
    def editar(self):
        #solicitar as novas informações
        print("Editando Dados do Livro...")
        return True
    
    def __del__(self):
        print("Executando o Destrutor da Classe...")
        return True

File: SIMauricio/JAVATestes/Livro.java
public class Livro {
    String titulo;
    String autor;
    String editora;
    int anoPublicacao;
    float preco;

    //Contrutor que NÃO recebe Parâmetros
    Livro(){

    }

    //Contrutor que RECEBE as informações por Parâmetro e atribui para os Atributos
    Livro(String titulo, String autor, String editora, int anoPublicacao, float preco){
        this.titulo = titulo;
        this.autor = autor;
        this.editora = editora;
        this.anoPublicacao = anoPublicacao;
        this.preco = preco;
    }

    boolean salvar(){
        //Validar os dados e salvar
        System.out.println("Salvando dados do Livro ...");
        return true;
    }
    
    boolean editar(){
        //solicitar as novas informações
        System.out.println("Editando Dados do Livro ...");
        return true;
    }    

  
}
